packages/firebase_remote_config/lib/src/remote_config.dart

```diff
@@ -25,17 +25,24 @@ class RemoteConfig extends ChangeNotifier {
   RemoteConfigSettings _remoteConfigSettings;
 
   DateTime get lastFetchTime => _lastFetchTime;
+
   LastFetchStatus get lastFetchStatus => _lastFetchStatus;
+
   RemoteConfigSettings get remoteConfigSettings => _remoteConfigSettings;
 
-  static Completer<RemoteConfig> _instanceCompleter = Completer<RemoteConfig>();
+  @visibleForTesting
```

Comment:
We're generally trying to avoid @visibleForTesting APIs since they clutter the public logs, though I don't see another solution at this point.

Reply:
Totally agree, I tried to find an alternative first before using it, but have to change it back somehow...

Reply:
@FirebaseExtended/invertase @collinjackson is there any other solution to be able to test for this edge case?